README.md

```diff
@@ -1 +1,87 @@
-# Restaurant-page
\ No newline at end of file
+# Restaurant-page
+
+## About the app
+
+> In this project I build a restaurant app where a user can buy nice food, this is done by navigating through menu and choose from available dishes, users can also make direct contact for orders.
+
+## Getting Started
+
+Enjoy app features by openning the link below in the blowser.
+
+### Have a copy
+
+## Setup
+
+To get started, you should first get this file in your local machine by cloning or forking this project or typing in your terminal
+
+> git clone https://github.com/elisha2kyakpo1/Restaurant-page.git
+
+> cd restaurant-page
+
+> git checkout restaurant-page
```

Comment:
There is no need to checkout any branch, since this readme file will go on the main branch after you merge all the changes, so since it will remain there forever, and all of your changes will be there, then there is no need to checkout any other branch because all of changes will be on that `main` branch. I would suggest you remove this step